recipes/metamobilepicker/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "MetaMobilePicker" %}
+{% set version = "0.7.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://gitlab.com/jkerkvliet/{{ name }}/-/archive/v{{ version }}/{{ name }}-v{{ version }}.tar.gz"
+  sha256: 6e2133bb60202db1f46e147bce3c86c35d281d218e1d19648e8c7be99463200c
+
+build:
+  number: 0
+  noarch: python
+  script: 
+    - "{{ PYTHON }} -m pip install . --ignore-installed --no-deps"
+
+requirements:
+  host:
+    - python ==3.10
+    - setuptools
+    - setuptools_scm
+    - setuptools_scm_git_archive
+    - pip
+  run:
+    - python ==3.10
```

Comment:
specifying the python version might wreck users' environment. Is this version a hard dependency?